Bindings/bindings.json

```diff
@@ -1474,6 +1474,14 @@
                     ]
                 },
                 {
+                    "name": "apiKey",
+                    "value": "string",
+                    "defaultValue": "SendGridApiKey",
```

Comment:
Maybe it's the case that existing apps with the AzureWebJobsSendGridApiKey will continue to work if they don't specify the setting in the binding? So to migrate, people can just fix their function code, but won't have to add a new app setting.